File: and1_1.py
# if에 and를 이용한 예제
input_name = input("이름을 입력해주세요.\n")
input_num = input("학번을 입력해주세요.\n")
name = "leejaejun"
num = "201521598"
if name == input_name and input_num == num:
    #name과 num이 동일하면 출력
    print("Hello!")
else:
    print("로그인에 실패했습니다")

File: and1_1.rb
#and로 통합한 예제
puts("아이디를 입력해주세요")
input_name = gets.chomp()
puts("비밀번호를 입력해주세요")
input_num = gets.chomp()
real_name = "egoing"
real_num = "11"
if real_name == input_name and real_num == input_num
    #name 과 num이 동일하면 출력
    puts("Hello!")
else
  puts("로그인에 실패했습니다")
end
